src/DependencyInjection/ContentApiExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\DependencyInjection;
+
+use Libero\PingController\PingController;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use function array_keys;
+
+final class ContentApiExtension extends Extension
+{
+    public function load(array $configs, ContainerBuilder $container) : void
+    {
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+
+        $loader->load('services.xml');
+
+        $configuration = new Configuration();
+
+        $config = $this->processConfiguration($configuration, $configs);
+
+        foreach ($config['services'] as $prefix => $service) {
+            $this->addService($prefix, $service, $container);
```

Comment:
Could do with clearer names, as we're talking about an API service and Symfony services.